lib/ex_unit/test/ex_unit_test.exs

```diff
@@ -510,4 +544,12 @@ defmodule ExUnitTest do
     output = capture_io(fn -> Process.put(:capture_result, ExUnit.Runner.run(opts, nil)) end)
     {Process.get(:capture_result), output}
   end
+
+  defp next_message_in_mailbox() do
+    receive do
+      msg -> msg
+    after
+      1 -> nil
```

Comment:
This should be zero. :)

Reply:
I must have missed that line in the documentation 😝